docs/partner-apps/bundleb2b/b2b-edition.md

```diff
@@ -0,0 +1,162 @@
+	'dashboard.container': '.container .page',
+}
+```
+
+For a list of B3 container names and their default placements, see the [BundleB2B Developer Guide](https://developer.bundleb2b.net/storefront/containers.html).
+
+### Customize styles
+
+You can customize the styling of many B3 elements using CSS.
+
+To modify the styling of a B3 module, first, insert `window.b3themeConfig.useStyles = {}` into your theme's `assets/js/global.js` file. Within `window.b3themeConfig.useStyles = {}`, create a property that uses the B3 element name as the key and one or more comma-separated CSS declarations as the value.
+
+Since the CSS is written in a JavaScript object, properties with two names, like `background-color`, must be written with camel case syntax: `backgroundColor: "lightblue"`
+
+When done, the object will resemble the following:
+
+```jsx
+window.b3themeConfig.useStyles = {
+/* B3 will use the specified styles for the "Trade Partner Application" button that is appended to the secondary navigation menu */
+      'tpa.entryButton': {
+        fontFamily: 'Karla,Arial,Helvetica,sans-serif',
+        fontSize: '1rem',
+        listStyle: 'none',
+        boxSizing: 'border-box',
+        lineHeight: 'inherit',
+        transition: 'color .15s ease',
+        display: 'block',
+        color: '#333',
+        fontWeight: 700,
+        padding: '1rem .78571rem',
+        textDecoration: 'none',
+        textTransform: 'uppercase',
+      },
+    };
+```
+
+For diagrams of B3 element names and their placements, see the [BundleB2B Developer Guide](https://developer.bundleb2b.net/storefront/containers.html).
+
+### Customize display text
+
+B3 allows you to customize display text for many of the application's elements, such as buttons, headers, and labels.
+
+To overwrite the default text that B3 renders, insert `window.b3themeConfig.useText = {}` into your theme's `assets/js/global.js` file. Within `window.b3themeConfig.useText = {}`, create a property for each element you would like to overwrite using B3 element names as keys and strings containing the new display text as values.
+
+When done, the object will resemble the following:
+
+```jsx
+window.b3themeConfig.useText = {
+/* B3 will now use the call to action "Place Quick Order" instead of the default "Quick Order Pad" for the button that is appended to the secondary navigation menu */
+ 'nav.button.quickOrderPad': 'Place Quick Order',
+}
+```
+
+For B3 element names and default text values, see the [BundleB2B Developer Guide](https://developer.bundleb2b.net/storefront/text.html).
+
+## Overwriting and injecting JavaScript
+
+B3 has lifecycle methods for many modules that allow you to inject custom JavaScript functions at different times during page render. Each supported module has the following four global keys:
+
+```jsx
+overwrite: false,
+callback() {},
+beforeMount() {},
+mounted() {},
```

Comment:
```suggestion
mounted() {}
```

Reply:
Do we need a comma after the last key?

Reply:
@ektravel It is not necessary, I included it because it was in the B3 documentation. They're supported in objects since ES5, but not JSON since it is based on pre-ES5 JS. I was curious what best practices are, so I looked it up. Some reasons one might use them are

> If you want to add a new property, you can add a new line without modifying the previously last line if that line already uses a trailing comma. This makes version-control diffs cleaner and editing code might be less troublesome.